frontend/src/components/recipePage.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import './recipePage.css'
+
+export default function Recipe({name, image, ingredients, instructions}) {
+
+    // state variable that stores ingredient within RecipePage component    
+    const [newIngredient, setNewIngredient] = React.useState('');
+    const [newInstruction, setNewInstruction] = React.useState('');
+
+    const addIngredient = () => {
+        console.log(newIngredient)
+        let ul = document.getElementById("ingredientList");
+        let random = document.createElement("li");
```

Comment:
This is a small issue, but try to use more descriptive variable names in the future. `ingredient `or `li` should work well.